x-pack/test/licensing_plugin/scenario.ts

```diff
@@ -67,6 +67,31 @@ export function createScenario({ getService, getPageObjects }: FtrProviderContex
       expect(response.body.trial_was_started).to.be(true);
     },
 
+    async startEnterprise() {
+      const response = await esSupertestWithoutAuth
+        .post('/_license/?acknowledge=true')
+        .send({
+          license: {
+            uid: '00000000-d3ad-7357-c0d3-000000000000',
+            type: 'enterprise',
+            issue_date_in_millis: 1577836800000,
+            start_date_in_millis: 1577836800000,
+            // expires 2022-12-31
+            expiry_date_in_millis: 1672531199999,
```

Comment:
The test will break in 2 years, but we don't have any dev license with a longer lifespan. Should I create an issue or write this down somewhere?

Reply:
🙈 We can schedule a slackbot reminder for 2022-12-01?

Reply:
We can store it in a separate file to import in a script to run validation. Not a blocker for the current PR.